app/views/teams/_form.html.erb

```diff
@@ -11,23 +11,21 @@
         <%= form.hidden_field :parent_team_id %>
       <% end %>
 
-      <%= render "shared/components/form_errors", resource: team %>
+      <%= render 'shared/components/form_errors', resource: team %>
 
       <div class="flex flex-col gap-4 items-start justify-center w-full">
-        <label class="text-sm flex flex-col">Upload team banner <span class="text-xs text-slate-grey font-light">(file size should be less than 1MB)</span></label>
+        <label class="text-sm flex flex-col">Upload team banner <span class="text-xs text-slate-grey font-light">(file size should be less than 1MB and Rsolution should be 1120 X 194 px)</span></label>
```

Comment:
Move this text to en.yml file and reference it here using I18.t function. 
"File size should be less than 1MB and recommended resolution is 1120 x194 pixels" for team banners